packages/wpcom.js/package.json

```diff
@@ -1,68 +1,48 @@
 {
-  "name": "wpcom",
-  "description": "Official JavaScript library for the WordPress.com REST API",
-  "version": "5.4.2",
-  "main": "build/index.js",
-  "author": "Automattic, Inc.",
-  "private": true,
```

Comment:
I think we should preserve this for now. This is only one of several PRs that will be modernising this, right?

Reply:
OK, I separated activating the package and using it in Calypso to the last commit in the branch. That way we can test it really works in Calypso and remove the commit before merging.

Reply:
Just to clarify, if the goal is to use the library within the monorepo, I don't think it needs to be unmarked as `private`.

I'm ok with making it public at the end of this PR, but it doesn't have to be done, if e.g. there are other breaking changes planned for subsequent PRs. For example, I would like to drop the dependency on `wpcom-xhr-request`, since it's only used as a default request handler. I would much rather have the burden of choosing a request handler resting solely on the consumer.

Reply:
> Just to clarify, if the goal is to use the library within the monorepo, I don't think it needs to be unmarked as `private`.

That's true, I totally missed that! Putting the `private` field back.